gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -41,6 +41,14 @@ def files(unit_extension)
       end
     end
 
+    def my_dirs
+      dirs = []
+      (SYSTEM_DIRS + USER_DIRS).each do |dir|
+        dirs << dir if @fs.fileExists?(dir)
+      end
+      dirs
```

Comment:
No I"ll do this
ALL_DIRS = (SYSTEM_DIRS + USER_DIRS) and then operate on ALL_DIRS in the two places.
And I do see what self.detected?(fs) does and the extreme similarity to existing_dirs after I change it.  Very similar but not the same.
Could be any_existing_dirs (with "any") and all_existing_dirs (with "select").